src/senaite/core/browser/contentmenu/view.py

```diff
@@ -41,12 +45,23 @@ def fiddle_menu_item(self, item):
         Unfortunately, this can not be done more elegant w/o overrides.zcml.
         https://stackoverflow.com/questions/11904155/disable-advanced-in-workflow-status-menu-in-plone
         """
+        def is_wf_item_visible(item):
+            """Checks if the WF items is visible or not
```

Comment:
I think you can achieve this by changing the "category" from the transition from "workflow" to anything else (e.g. "listing"). Via ZMI:

![Captura de 2022-08-17 17-00-47](https://user-images.githubusercontent.com/832627/185173940-53d79528-473a-4d7b-83b3-406682f6d1c7.png)

Or in transition definition:

```xml
<transition transition_id="listing_multi_results" title="Multi Results" new_state="" trigger="USER"
              allowed="False" before_script="" after_script="" i18n:attributes="title">
    <action url="" category="listing" icon="">Multi Results</action>
    ...
```

Not sure about side effects though

Reply:
Ok, that might work as well, let me give it a try. Maybe I try first how a default Plone site renders a transition with this category in...

Reply:
It looks like any other category than `workflow` is skipped already:
https://github.com/plone/plone.app.contentmenu/blob/master/plone/app/contentmenu/menu.py#L840